ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/MiningParametersMetrics.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Hyperledger Besu Contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.ethereum.core;
+
+import org.hyperledger.besu.metrics.BesuMetricCategory;
+import org.hyperledger.besu.plugin.services.MetricsSystem;
+
+public class MiningParametersMetrics {
+  public static final String MIN_GAS_PRICE_GAUGE = "min_gas_price";
+  public static final String MIN_PRIORITY_FEE_GAUGE = "min_priority_fee";
+  private final MetricsSystem metricsSystem;
```

Comment:
it seems not necessary to keep this reference, since everything is done in the constructor